addons/l10n_co_pos_report/__manifest__.py

```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+{
+    'name': 'Colombian - Point of Sale Report',
+    'version': '1.0',
+    'description': """Colombian - Point of Sale""",
```

Comment:
report will surely be confusing with the reports in enterprise. Maybe pos_details is a better one